src/pyspiffe/svid/jwt_svid.py

```diff
@@ -1,19 +1,156 @@
-from typing import List
 from src.pyspiffe.spiffe_id.spiffe_id import SpiffeId
+from src.pyspiffe.bundle.jwt_bundle.jwt_bundle import JwtBundle
+from src.pyspiffe.exceptions import JwtSvidError
+import datetime
+from calendar import timegm
+import jwt
+from jwt import exceptions
+
+EMPTY_TOKEN_ERROR = 'Token cannot be empty.'
+INVALID_INPUT_ERROR = 'Invalid input {}.'
+MISSING_X_ERROR = 'Token is missing {}.'
+AUDIENCE_NOT_MATCH_ERROR = 'Audience does not match payload[aud] in Token.'
+SIGNATURE_EXPIRED_ERROR = 'Token signature has expired.'
 
 
 class JwtSvid(object):
     """
-    Represents a SPIFFE X.509 SVID.
+    Represents a SPIFFE JWT SVID as defined in the SPIFFE standard.
+    See <a href="https://github.com/spiffe/spiffe/blob/master/standards/JWT-SVID.md">https://github.com/spiffe/spiffe/blob/master/standards/JWT-SVID.md</a>.
+    """
+
+    _required_claims = ['aud', 'exp', 'sub']
 
-    Contains a SPIFFE ID, a private key and a chain of X.509 certificates.
+    def __init__(
+        self, spiffeId: SpiffeId, audience: [], expiry: datetime, claims: {}, token: str
+    ):
+        self.spiffeId = spiffeId
+        self.audience = audience
+        self.expiry = expiry
+        self.claims = claims
+        self.token = token
 
-    :param spiffe_id: SPIFFE ID of the X509-SVID.
-    :param private_key: Audience is the intended recipients of JWT-SVID as present in the 'aud' claim.
-    :param chain: Expiration time of JWT-SVID as present in 'exp' claim.
     """
+    Parses and validates a JWT-SVID token and returns an instance of a JwtSvid with a SPIFFE ID parsed from the 'sub', audience from 'aud', 
+    and expiry from 'exp' claim. The JWT-SVID signature is not verified.
+    
+    Ags:
```

Comment:
nit: `Args:`, also `Returns:` is missing

Reply:
Done, thanks!